docs/cluster-deployment/stripe.rst

```diff
@@ -1,134 +1,101 @@
+deployment screen. Here you can configure the name, region, compute, storage,
+and the number of deployed nodes.
 
-First, select a `region`_. Your cluster will be deployed in this region.
-
-.. image:: ../_assets/img/stripe-regions.png
-   :alt: Region selection
-
-.. NOTE::
-    Your choice of region may affect the price of the cluster. Refer to the
-    price total indicated at the bottom right of the screen.
+.. image:: ../_assets/img/deployment-config.png
+   :alt: CrateDB Cloud Console Deployment Configuration
 
+Region
+------
 
-Configure cluster
------------------
+First, after naming your cluster, select a `region`_ where your cluster should
+be deployed. Currently, we offer AWS and Azure regions. If you'd like to 
+request a new region, there is a form present. Simply click the "Request a new
+region" link, and the form will appear:
 
-Next, go through the cluster configuration process. On the left-hand side, you
-can choose a `subscription plan`_. As you select each subscription plan, you
-will see that the values for CPU, RAM, and (minimum) storage per node change
-accordingly in the middle panel that shows the node specification.
+.. image:: ../_assets/img/deployment-region-request.png
+   :alt: CrateDB Cloud Console Deployment New Region Request
 
-.. image:: ../_assets/img/stripe-config.png
-   :alt: Cluster configuration
+If you are deploying an :ref:`Edge <edge>` cluster, you can choose your
+custom region by clicking the "Add a custom edge region" link.
 
-Each subscription plan comes with a preconfigured compute value per node. The
-compute capacity of your cluster can be scaled horizontally by adding nodes. 
-To do this, use the slider under the 'Cluster scale' panel. This overview  
-shows the current CPU and RAM values of your cluster, which are simply the
-compute values of the plan you have chosen multiplied by the number of nodes.
+Compute
+-------
 
-CrateDB Cloud separates storage configuration from compute configuration. You
-can separately configure the desired storage for your cluster by using the
-dropdown menu under 'Storage' in the 'Node specification' panel. The range of
-eligible storage sizes varies by subscription plan. The storage you have  
-chosen for your cluster is also reflected in the cluster overview in the
-right-hand
-panel.
-
-.. NOTE::
-    The storage capacity of a cluster cannot currently be lowered in the
-    CrateDB Cloud Console once it has been deployed.
+In the Compute section, you should choose a `subscription plan`_ that suits
+your needs and use case. Plans consist of a per-node CPU and RAM allocation.
 
 Shared-resources clusters (CR0)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The Shared-resource clusters are a special category of clusters that allow
-better utilisation of resources. These clusters share compute and storage
+better utilization of resources. These clusters share compute and storage
 resources with other clusters in this category. Because of this, they offer
```

Comment:
This wasn't included in your changes but this sentence is missing an 'a'. So maybe where while you are here this should be 


Because of this, they offer a more cost-effective solution